app/src/main/java/com/karumi/androidanimations/sharedelements/SharedElementsExerciseTransitionFragment.kt

```diff
@@ -57,7 +56,14 @@ class SharedElementsExerciseTransitionFragment : BaseFragment() {
 
         val enterSharedElementTransition
             get() = TransitionSet()
-                .addTransition(GlobalPositionTransition().accelerated.withArc)
```

Comment:
I find it interesting that it looks the same as the exit transition xD

Reply:
Makes sense! With the transitions API you only specify how the change between the values should animate in terms of timing and what values should animate but not the values 😃